root/enry.py

```diff
@@ -0,0 +1,68 @@
+import ast
+from collections import defaultdict
+import json
+import os
+import platform
+import subprocess
+
+from tqdm import tqdm
+
+from extract import logger
+from treesitter import queries, save_identifiers
+
+unparseable_languages = set()
+
+
+def extract_languages(repo_path: str):
+    """
+    Traverse repository directory, for each programming language file create a json using enry,
+    Write json to jsonl file
+    :param repo_path: path of extracted repository
+    """
+    path = os.walk(repo_path)
+    file_dict = defaultdict(dict)
+    with open("../used_jsons/" + repo_path.split("/")[-1] + ".json", "w") as json_file:
+        for root, _, files in tqdm(path):
+            for file in files:
+                language_json = get_enry_result(root + "/" + file)
+                if language_json["language"] != "" and not language_json["vendored"]:
+                    add_file_language_info(root, language_json, file_dict)
+        json_file.write(json.dumps(file_dict))
+    return file_dict
+
+
+def get_enry_result(file_path: str) -> dict:
+    """
+    Get language, number of lines and vendor info for a file
+    :param file_path: path to file
+    :return: dictionary with language and number of lines in file
+    """
+    enry_result = subprocess.check_output(
+        [".././enry.exe" if platform.system() == "Windows" else "./enry", "-json",
```

Comment:
hardcode! Let's specify paths to `enry` - because user have to install `enry` manually